osbrain/address.py

```diff
@@ -238,14 +272,17 @@ class AgentAddress():
         Agent kind.
     role : AgentAddressRole
         Agent role.
+    serializer : AgentAddressSerializer
+        Agent serializer.
     """
-    def __init__(self, transport, address, kind, role):
+    def __init__(self, transport, address, kind, role, serializer=None):
```

Comment:
I would rather make `serializer` parameter non-optional (default should be loaded in Agent's implementation).